runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -1954,6 +1954,22 @@ private static WorldPoint getViggoraLocation(ClueScrollPlugin plugin)
 		}
 	}
 
+	private static String getViggoraLocationString(ClueScrollPlugin plugin)
+	{
+		int varb = plugin.getClient().getVarbitValue(Varbits.VIGGORA_LOCATION);
+		switch (varb)
+		{
+			case 1:
+				return "the Rogue's Castle";
```

Comment:
"the" could be moved into the clue's text instead of being in each case here, but that's not a big deal

Reply:
If he ever moves in the future the location might not include a `the`